.storybook/src/grid-layer.stories.ts

```diff
@@ -47,8 +47,8 @@ const createEventObj = (elm: any) => {
     .range([0, 500]);
 
   return {
-    xscale: xscale.copy(),
-    yscale: yscale.copy(),
+    xScale: xscale.copy(),
```

Comment:
Should the other xscale be changed too?

Reply:
Given that this is a storybook example and the the variable is a copy, I don't think it's necessary. But we should probably discuss naming conventions of variables and functions to be consistent.